ballista/rust/core/src/serde/physical_plan/to_proto.rs

```diff
@@ -132,6 +132,7 @@ impl TryInto<protobuf::PhysicalPlanNode> for Arc<dyn ExecutionPlan> {
                 JoinType::Inner => protobuf::JoinType::Inner,
                 JoinType::Left => protobuf::JoinType::Left,
                 JoinType::Right => protobuf::JoinType::Right,
+                JoinType::Full => protobuf::JoinType::Right,
```

Comment:
This looks like it is a typo perhaps

```suggestion
                JoinType::Full => protobuf::JoinType::Full,
```